test/fixtures/engine/engine-dependencies/app1/terragrunt.hcl

```diff
@@ -0,0 +1,9 @@
+include "root" {
+  path = find_in_parent_folders("root.hcl")
+}
+
+engine {
```

Comment:
I think the engine block can be extracted to the parent `root.hcl` to avoid duplications